frontend/openchat-shared/src/utils/chat.ts

```diff
@@ -44,10 +44,9 @@ export function userIdsFromEvents(events: EventWrapper<ChatEvent>[]): Partitione
         }
         switch (e.event.kind) {
             case "message":
+                userIds.add(e.event.sender);
```

Comment:
All users now get added to `userIds` and the webhooks are excluded later.
This is because is some scenarios it is impossible to detect if a user is a webhook or not, eg. if a webhook message is replied to.